roles/inherit_kerberos/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: "Install client packages on Red Hat based distributions"
+  ansible.builtin.dnf:
+    name:
+      - "krb5-workstation"
+      - "krb5-libs"
+    state: present
+  when: ansible_os_family == "RedHat"
+
+- name: "Install client packages on Debian based distributions"
+  ansible.builtin.apt:
+    name: "krb5-user"
+    state: present
+  when: ansible_os_family == "Debian"
```

Comment:
you culd use `ansible.builtin.package` which works across distros, and load the correct list of packages using a vars file.

https://github.com/theforeman/forklift/blob/12c6385b216e77dc4565d5ef060deae3745cd24e/roles/bats/tasks/main.yml#L3
https://github.com/theforeman/forklift/blob/12c6385b216e77dc4565d5ef060deae3745cd24e/roles/bats/vars/RedHat.yml#L2

Not saying you *have* to, but something to consider :)